CONTRIBUTING.md

```diff
@@ -30,37 +33,86 @@ Regular contributor meetings are held on Slack, see [`docs/contributor-meetings.
 
 ## Contribution workflow
 
-This is a rough outline of how to prepare a contribution:
 
-- Find an [issue](https://github.com/weaveworks/eksctl/issues) to work on. If you are a new contributor
-take a look at issues marked with [good first issue](https://github.com/weaveworks/eksctl/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
-- Create a topic branch from where you want to base your work (usually branched from master).
-- Make commits of logical units.
-- Make sure your commit messages are in the proper format (see below).
-- Push your changes to a topic branch in your fork of the repository.
-- If you changed code:
-   - add automated tests to cover your changes. See the [az](https://github.com/weaveworks/eksctl/tree/master/pkg/az) package for a good example of tests.
-- Submit a pull request to the original repository.
+#### 1. Set up your Go environment
 
-If your PR is a work in progress then make sure you prefix the title with `WIP: `. This lets everyone know that this is still being worked on. Once its ready
-remove the `WIP: ` title prefix and where possible squash your commits.
+This project is written in Go. To be able to contribute you will need a working Go installation. You can check the 
+[official installation guide](https://golang.org/doc/install). Make sure your `GOPATH` and `GOBIN` 
+[environment variables](https://github.com/golang/go/wiki/SettingGOPATH) are set correctly.
 
-## How to build and run the project
+Here are some quick start steps you can follow:
 
+Select a folder path for your go projects, for example `~/eksctl`
 ```bash
-make build
-./eksctl get clusters
+mkdir ~/eksctl
+cd ~/eksctl
+export GOPATH="$(pwd)"
+export GOBIN=$GOPATH/bin
 ```
 
-## How to run the test suite
+#### 2. Fork and clone the repo
 
-You can run the unit tests by simply doing
+Clone the repo using go:
 
 ```bash
+go get -d github.com/weaveworks/eksctl
```

Comment:
Some people complained that it produces an error (albeit it closes successfully)...

```$ go get -d github.com/weaveworks/eksctl
package github.com/weaveworks/eksctl: no Go files in /Users/ilya/eksctl/src/github.com/weaveworks/eksctl
$ echo $?
1
$ rm -rf src
$ go get -d github.com/weaveworks/eksctl/...
$ echo $?
0
$
```

```suggestion
go get -d github.com/weaveworks/eksctl/...
```

Reply:
yeah, I got a bit confused too. This is better. Done

Reply:
the `...` suffix means "look in all subdirectories", btw